src/components/header/style.module.less

```diff
@@ -389,6 +388,7 @@
 	height: 100%;
 	align-items: center;
 	border: none;
+	color: #eee;
```

Comment:
Text in the nav area is `#eee`, the social links are `#f0f0f0`, and the dropdown arrows were `#fff`.

I just picked the nav link text to match